spectron/core/publishing/app-publishing.func.ts

```diff
@@ -59,7 +59,8 @@ describe("app publishing", () => {
 
                                 return (appID: string) => {
 
-                                    if (appID.startsWith("test-user/test-project/test-app-publish") && $callCount > 1) {
+                                    $callCount++;
```

Comment:
Doesn't incrementing `$callCount` manually defeat the purpose of it?

Reply:
$callCount incrementation did not work earlier - the problem was that it only got called once, every time a new MockSBGClient was constructed. This means the $callCount would always evaluate to 1.

What I can probably do is insert the 'callCount++;' into the function implicitly, but I would rather all of us discuss this and decide which direction to go in as far as writing tests is concerned. For now, just leave it as is and we will change if necessary.